app/client/src/components/Post.jsx

```diff
@@ -66,22 +67,52 @@ const Post = (props) => {
   const handlePostClick = () => navigate(`/post/${post._id}`);
   const handleCommunityClick = (title) => navigate(`/community/${title}`);
 
+  // Create preview if post message is longer than 250 characters
+  let message = post.message;
+  if (message.length > 250) {
+    message = `${message.substr(0, 249)}...`;
+  }
+
+  // Convert to local time
+  let createdOn =
+    moment(moment.utc(post.createdOn, "MMMM Do YYYY, h:mm:ss a").toDate())
+      .local()
+      .format("MMMM Do YYYY, h:mm:ss a") || "";
+  // Remove milliseconds
+  createdOn = `${createdOn.substr(0, createdOn.length - 6)} ${createdOn.substr(
+    createdOn.length - 2,
+    createdOn.length
+  )}`;
+  const splitCreatedOn = createdOn.split(",");
+
+  const today = moment().format("MMMM Do YYYY");
+  const yesterday = moment().subtract(1, "days").format("MMMM Do YYYY");
+
+  if (splitCreatedOn[0] === today) createdOn = `Today${splitCreatedOn[1]}`;
+  if (splitCreatedOn[0] === yesterday)
+    createdOn = `Yesterday${splitCreatedOn[1]}`;
+
   return (
-    <div key={post._id}>
+    <div
+      key={post._id}
+      style={{ "margin-bottom": "15px", "background-color": "transparent" }}
```

Comment:
Whenever you want to style components, use the mui sx prop. Using sx is the recommended method to use for styling and it is theme-aware.

Reply:
I just thought sx was only for react components. I'll change it to keep consistency.

Reply:
I just tested and sx doesn't work on <div>. We could change the div's to <Box> for now I'll leave it as is.